Bottom: c89745d7513e3361c5fa8f591c0d916333fd110f
Top:    9d5e87ec3982847e89b8102eccac7589d928ca56
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:36:43 +0000

Refresh of w32-add-recipe-for-freetype

---

diff --git a/w32/build-all.sh b/w32/build-all.sh
index 36e45d9..04aec82 100755
--- a/w32/build-all.sh
+++ b/w32/build-all.sh
@@ -9,6 +9,7 @@ jpeg
 zlib
 libpng
 tiff
+freetype
 pixman
 libiconv
 gettext
